front/app/components/admin/PostManager/components/PostTable/index.tsx

```diff
@@ -78,8 +79,8 @@ const Container = styled.div`
   }
 `;
 
-export const TableHeaderCellText = styled.span`
-  font-weight: 600;
+const StyledTable = styled(Table)`
```

Comment:
Really needed? Could we use our base font size?

Reply:
Yeah screw it